horovod/common/operations.cc

```diff
@@ -694,6 +694,36 @@ bool horovod_is_initialized() {
   return horovod_global.initialization_done;
 }
 
+bool horovod_start_timeline(const char* file_name, bool mark_cycles) {
+  if (!horovod_global.initialization_done) {
+    return false;
+  }
+
+  bool is_coordinator = horovod_global.controller->IsCoordinator();
+  if (is_coordinator) {
+    horovod_global.timeline.Initialize(file_name, horovod_global.controller->GetSize());
```

Comment:
i guess for safety you should check if not inited then init. 
will this cause timeline to get reinited if starttimeline is called twice ?